app/services/scholars_archive/triple_powered_service.rb

```diff
@@ -73,15 +72,13 @@ def predicate_labels(graph)
           .query(predicate: predicate)
           .select { |statement| !statement.is_a?(Array) }
           .map(&:object)
-          .select { |value| value.respond_to?(:language) ? value.language.in?(%i[en en-us]) : true }
```

Comment:
We still want to prefer english labels if we can. So maybe we take the above `#map` call and save that as `labels[predicate.to_s] = all_labels.map(&:to_s)`
Then we can run the `#select` and see if it has anything in it. If it does, re-set `labels[predicate.to_s] = eng_labels.map(&:to_s)`?